forem/forem
# Cleanup markup of editor guide

## What type of PR is this? (check all applicable)


## Description

I tried to cleanup a bit the Markdown's editor guide page as its HTML and accessibility score were subpar.
I haven't solved everything (especially in the domain of semantic tags) but I think it's a bit better than before.

## QA Instructions, Screenshots, Recordings

Load the branch, visit http://localhost:3000/p/editor_guide and check everything is okay.

In terms of UI:

**before**

![Screenshot 2021-04-24 at 13-50-57 Editor Guide - DEV Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/115957739-70737880-a504-11eb-8ac9-ce0b755095b4.png)


**after**

![Screenshot 2021-04-24 at 13-51-23 Editor Guide - DEV(local) 🌱](https://user-images.githubusercontent.com/146201/115957743-75d0c300-a504-11eb-9229-a3d2ea2db5d8.png)


In terms of accessibility score:

**before**

![Screenshot 2021-04-24 at 10 50 52 AM](https://user-images.githubusercontent.com/146201/115957630-e9260500-a503-11eb-9c9c-356d05889cc6.png)

**after**

![Screenshot 2021-04-24 at 10 50 24 AM](https://user-images.githubusercontent.com/146201/115957646-f7742100-a503-11eb-9a06-b1ab00478a0f.png)

## Added tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
